src/core/data/actions/index.js

```diff
@@ -5,10 +5,20 @@
 import { newRequestSequence } from 'redux-reqseq';
 import type { RequestSequence } from 'redux-reqseq';
 
+const FETCH_ATLAS_DATA_SET_DATA :'FETCH_ATLAS_DATA_SET_DATA' = 'FETCH_ATLAS_DATA_SET_DATA';
+const fetchAtlasDataSetData :RequestSequence = newRequestSequence(FETCH_ATLAS_DATA_SET_DATA);
```

Comment:
This might not be necessary. Let's add it to the list of things to talk about.

Reply:
yeah, right now I don't think we even display the search on atlas data sets.

Reply:
Correct. We can't actually search atlas, so we decided against showing atlas data in the app, at least for now. Should we even bring in all the `fetchAtlasDataSetData` stuff?

Reply:
I can remove it!